packages/core/src/Tooltip/Tooltip.spec.js

```diff
@@ -55,4 +55,23 @@ describe('Tooltip', () => {
     ).toJSON()
     expect(json).toMatchSnapshot()
   })
+
+  describe('deprecated prop types', () => {
+    const consoleError = console.error
+    beforeEach(() => (console.error = jest.fn()))
+    afterEach(() => (console.error = consoleError))
```

Comment:
Just for consistency, we might want to go with something like this where applicable:
```javascript
    let consoleError
    beforeEach(() => {
      consoleError = console.error
      console.error = jest.fn()
    })
    afterEach(() => (console.error = consoleError))
```

I'm not sure it makes a difference, but I generally like keeping all assignments inside of a before/afterEach just to be safe.

Reply:
[Updated!](https://github.com/priceline/design-system/pull/900/files/167e8cedaa0b0eb52fcce4eded81b63f420adc8b..8452cb48afe66bbd483f8a42d0fbbc4e428d5744)